secure_message/repository/modifier.py

```diff
@@ -91,12 +91,14 @@ def del_draft(draft_id):
         del_draft_status = "DELETE FROM securemessage.status WHERE msg_id='{0}' AND label='{1}'".format(draft_id, Labels.DRAFT.value)
         del_draft_event = "DELETE FROM securemessage.events WHERE msg_id='{0}'".format(draft_id)
         del_draft_inbox_status = "DELETE FROM securemessage.status WHERE msg_id='{0}' AND label='{1}'".format(draft_id, Labels.DRAFT_INBOX.value)
+        del_actors = "DELETE FROM securemessage.actors where msg_id='{0}'".format(draft_id)
         del_draft_msg = "DELETE FROM securemessage.secure_message WHERE msg_id='{0}'".format(draft_id)
 
         try:
             db.get_engine(app=db.get_app()).execute(del_draft_status)
             db.get_engine(app=db.get_app()).execute(del_draft_inbox_status)
             db.get_engine(app=db.get_app()).execute(del_draft_event)
+            db.get_engine(app=db.get_app()).execute(del_actors)
             db.get_engine(app=db.get_app()).execute(del_draft_msg)
 
         except Exception as e:
```

Comment:
Probably worth changing this to logger.exception while we are at it.

Reply:
done